SELECTOR_INFO.md

```diff
@@ -273,4 +273,445 @@ Selector Data: "menuitem[data-l10n-id='downloads-cmd-clear-downloads']"
+Selector Data: library-button
+Description: Library button
+Location: Toolbar after adding it from customize
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: library-history-submenu-button
+Selector Data: appMenu-library-history-button
+Description: Library history submenu button
+Location: Inside the library button in the toolbar
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: bookmarks-type-dropdown
+Selector Data: editBMPanel_folderMenuList
+Description: Bookmark location
+Location: Bookmark panel
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: bookmarks-type-dropdown-other
+Selector Data: editBMPanel_unfiledRootItem
+Description: Bookmark location - Other bookmarks
+Location: Bookmark panel
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: other-bookmarks
+Selector Data: OtherBookmarks
+Description: Other bookmarks option
+Location: Bookmark panel
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: other-bookmarks-popup
+Selector Data: OtherBookmarksPopup
+Description: Other bookmarks submenu
+Location: Bookmark panel
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: new-bookmark-name-field
+Selector Data: editBMPanel_namePicker
+Description: New bookmark name field
+Location: Bookmark panel
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: new-bookmark-url-field
+Selector Data: editBMPanel_locationField
+Description: New bookmark url field
+Location: Bookmark panel
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: new-bookmark-tags-field
+Selector Data: editBMPanel_tagsField
+Description: New bookmark tags field
+Location: Bookmark panel
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: new-bookmark-keyword-field
+Selector Data: editBMPanel_keywordField
+Description: New bookmark keyword field
+Location: Bookmark panel
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: bookmark-dialog
+Selector Data: ookmarkpropertiesdialog
+Description: Bookmark dialog
+Location: Bookmark panel
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: bookmark-accept-button
+Selector Data: button[dlgtype='accept']
+Description: Bookmark accept button
+Location: Bookmark panel
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: bookmark-iframe
+Selector Data: browser[class='dialogFrame']
+Description: Add bookmark advanced
+Location: Add bookmark from bookmarks sidebar
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: bookmark-robots
+Selector Data: menuitem[label='Robots 2']
+Description: A bookmark named robots
+Location: A bookmark in other bookmarks
+Path to .json: modules/data/navigation.components.json
+```
+```
+Selector name: bookmark-current-tab
```

Comment:
Should we move this selector (not just here, but also in the .json files) to PanelUI?

Reply:
I'm using this locator in a method in browser_object_navigation, if I move the locator I'm not sure how could I access the locator.

Reply:
That's fine for now, then. We may move it later....